From 74f52e791bef92ed4a5191494ae86e2e40d2ca17 Mon Sep 17 00:00:00 2001
From: Olivier Moysan <olivier.moysan@foss.st.com>
Date: Fri, 2 Jun 2023 11:43:39 +0200
Subject: [PATCH 0216/1141] arm64: defconfig: add audio configs in addons
 fragment

Add Audio peripherals I2S, SAI, SPDIRX and wolfson audio codec WM8994
config flags in addons fragment.

Signed-off-by: Olivier Moysan <olivier.moysan@foss.st.com>
Change-Id: I97032bb64221090c02f1d7229fa48bfa245eb1f8
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/308824
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 8 ++++++++
 1 file changed, 8 insertions(+)

--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -11,6 +11,7 @@ CONFIG_IIO_STM32_LPTIMER_TRIGGER=m
 CONFIG_IIO_STM32_TIMER_TRIGGER=m
 CONFIG_MFD_STM32_LPTIMER=m
 CONFIG_MFD_STM32_TIMERS=m
+CONFIG_MFD_WM8994=m
 CONFIG_NET_ACT_POLICE=m
 CONFIG_NET_ACT_SKBEDIT=m
 CONFIG_NET_ACT_VLAN=m
@@ -29,11 +30,18 @@ CONFIG_RTC_DRV_STM32=y
 CONFIG_SENSORS_MR75203=y
 CONFIG_SERIAL_STM32=y
 CONFIG_SERIAL_STM32_CONSOLE=y
+CONFIG_SND_SOC_STM32_I2S=m
+CONFIG_SND_SOC_STM32_MDF=m
+CONFIG_SND_SOC_STM32_SAI=m
+CONFIG_SND_SOC_STM32_SPDIFRX=m
+CONFIG_SND_SOC_WM8994=m
 CONFIG_SPI_STM32=m
 CONFIG_STM32_ADC=m
 CONFIG_STM32_ADC_CORE=m
 CONFIG_STM32_DMA3=y
 CONFIG_STM32_LPTIMER_CNT=m
+CONFIG_STM32_MDF_ADC=m
+CONFIG_STM32_MDF_CORE=m
 CONFIG_STM32_TIMER_CNT=m
 CONFIG_STMMAC_ETH=y
 CONFIG_STMMAC_PLATFORM=y
